source/_integrations/aqara.markdown

```diff
@@ -0,0 +1,57 @@
+---
+title: Aqara
+description: Instructions on how to set up the aqara devices within Home Assistant..
+ha_release: 0.0.1
+ha_category:
+  - Binary Sensor
+ha_iot_class: Cloud Push
+ha_config_flow: true
+ha_codeowners:
+  - '@wumao'
+  - '@chenliuyang1989'
+ha_domain: aqara
+ha_platforms:
+  - binary_sensor
+ha_integration_type: integration
+---
+
+Aqara integration allows users to integrate binary devices and scenes on the Aqara Home app into Home Assistant. After the account authorization, the supported devices and scenes will be automatically synchronized to the Home Assistant.
+
+## Configuration of the Aqara IoT Platform
+
+### PREREQUISITES
+
+- You will need to create an account in the Aqara Home app.
+- Your devices also need first to be added in the Aqara Home app.
+
+### ADD DEVICES BY AQARA HOME APP
```

Comment:
The Add Devices to App section may not be required as it isn't included on other integrations such as Xiaomi Miio or Tuya.